### DISCUSSION TITLE
Cant add cloud resource - wrong s3 endpoint

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#6847] [Q&A]
Hi.
I cant add cloud resource (operator v5.9.0). It gives me an error: "Please enter a valid s3 compatible endpoint". I try with https, without https... the result is always the same.
I would like to connect to our private s3 storage.

![image](https://user-images.githubusercontent.com/5565897/147771125-ca7f42e2-1c45-4630-b002-c3cb8c47d0a8.png)

**Comment by dkoci** [comment]
Anyone?

**Comment by jackyalbo** [comment]
The core pod probably doesn't have access to this IP address. Check if it works when you set the IP to Amazon's. And make sure you have internet connection. We don't really actively support the UI, but to the best of my knowledge this area suppose to work.

**Reply by dkoci** [reply]
I dont think this is the issue, because everything works fine in 5.8.0 version.
